### DISCUSSION TITLE
How do you test things like registration with email?

### DISCUSSION THREAD

**Discussion by romanr** [fzyzcjy/flutter_convenient_test#403] [General]
What is the best practice to test email-based registration that requires user to click the link in email?

**Comment by fzyzcjy** [comment]
I guess one way is to mock it.

**Reply by romanr** [reply]
Registration with email is pretty common scenario. I imagine around 100% of projects may use it. Anyone can share their way of testing it?

**Reply by fzyzcjy** [reply]
I guess you can google about this and there will be tutorials; also asking chatgpt or claude or etc may also provide some code for you.
